gradio/interface.py

```diff
@@ -110,7 +110,7 @@ def __init__(
         height=None, 
         width=None, 
         allow_screenshot: bool = True, 
-        allow_flagging: bool = None, 
+        allow_flagging: Optional[str] = None, 
```

Comment:
instead of optional[string] with default none, why not just a string with default "never".

Reply:
So that we can refer back to the environmental variable if none is specified through parameters